github_connector/models/github_organization_serie.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class GithubOrganizationSerie(models.Model):
+    _name = 'github.organization.serie'
```

Comment:
For the OCA, the organization series are : 
- 5.0
- 6.0
- 6.1
- 7.0
- 8.0
- 9.0
- 10.0

It is usefull for two things :   
- As this module download all the code source, it should ignore some branches. So all the branches with a name not in a serie will be ignored.
- To have this order of the series and affect correctly a commit to a serie. For exemple, if a commit is done on the 8.0 and so present in 9.0 and 10.0, it will be affected to the 8.0 serie.